io/zenoh-transport/src/unicast/establishment/accept.rs

```diff
@@ -687,7 +682,7 @@ pub(crate) async fn accept_link(link: LinkUnicast, manager: &TransportManager) -
     let iack_out = {
         let mut state = State {
             transport: StateTransport {
-                batch_size: manager.config.batch_size.min(batch_size::UNICAST).min(mtu),
```

Comment:
I'd rather silent this warning. It's convenient to be able to change a constant in the protocol and having all the changes reflected in the overall transport.